domains/rahil.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "rahilkzi",
+    "email": "rahilkazi66@gmail.com",
+    "discord": "rahilkzi"
+  },
+  "record": {
+    "CNAME": "cname.vercel-dns.com"
```

Comment:
Unable to access site:

![image](https://github.com/user-attachments/assets/1a6d49ee-ef5d-433c-979d-caa5cb5edf1b)

Please send preview/fix site.

Reply:
![Screenshot 2024-10-24 120133](https://github.com/user-attachments/assets/b77d4567-3c15-4062-871b-89a4334b00a1)